docs/usage/configuration-options.md

```diff
@@ -2043,6 +2043,7 @@ Consider using instead `matchCurrentValue` if you wish to match against the raw
   ]
 }
 ```
+The syntax of the SemVer range must follow the [versioning scheme](https://docs.renovatebot.com/modules/versioning/#supported-versioning) scheme used by the [manager](https://docs.renovatebot.com/modules/manager/#supported-managers) which discovered the configured package.
```

Comment:
What's wrong there's no output at https://github.com/renovatebot/renovate/actions/runs/4689904356/jobs/8312256317?pr=21485

Reply:
Why is it not outputting the expected contents?
What if I don't have a node.js dev env set up?

I'm trying
```
npm install
npm run prettier
```
but if it fails I'll just assume the lines are long and wrap them.

Reply:
Wow, that was way harder than it needed to be, a missing empty line. And then fighting with `.husky/pre-commit` hook because I don't have yarn installed. In the end the only way I was able to commit was on GitHub UI...

Reply:
How do I remove the git hook `npm install` added?

Reply:
I'm an Android / Java engineer, for a few .md changes I don't want to set up a whole new VM. 🙄

Reply:
> How do I remove the git hook npm install added?

Found it: `.git/config > hookspath`, side effects everywhere :)